src/editors/content/learning/XrefEditor.tsx

```diff
@@ -54,30 +55,33 @@ export default class XrefEditor
     this.onChangeTarget = this.onChangeTarget.bind(this);
     this.onChangePage = this.onChangePage.bind(this);
     this.onToggleTargetPage = this.onToggleTargetPage.bind(this);
-    this.state = { targetIsPage: props.model.idref === props.model.page };
+    this.state = {
+      targetIsPage: props.model.idref === props.model.page,
+      loadingTarget: true,
```

Comment:
This state variable is not being used - you can remove it

Reply:
done